pipeline/mist.go

```diff
@@ -75,6 +84,19 @@ func (m *mist) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	return ContinuePipeline, nil
 }
 
+func isVideo(source string) (bool, error) {
+	header, err := clients.GetArweaveHeaders(source)
+	if err != nil {
+		return false, err
+	}
+	contentType := header.Get("Content-Type")
+	mediaType, _, err := mime.ParseMediaType(contentType)
+	if err != nil {
+		return false, err
```

Comment:
@thomshutt do you know if we're ok to require the content-type header or should we fail open instead?

Reply:
I definitely don't think we should rely on it being set correctly

Reply:
yeah make sense, reverted that bit 👍